tapir/shifts/tests/test_shift_template_duplicate.py

```diff
@@ -75,8 +75,14 @@ def test_shiftTemplateDuplicateForm_duplicateSingleTemplate(self):
             follow=True,
         )
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(ShiftTemplate.objects.count(), 2)
+        self.assertEqual(2, ShiftTemplate.objects.count())
         for template in ShiftTemplate.objects.all():
+            self.assertEqual(
+                template.start_date.strftime("%Y%m%d"),
+                shift_template.start_date.strftime("%Y%m%d"),
+            )
+            self.assertEqual(template.start_time, shift_template.start_time)
+            self.assertEqual(template.end_time, shift_template.end_time)
```

Comment:
We can improve the names by specifying that `shift_template` is the source, and that `template` is the duplicate. 
Also, looping through all templates is a bit weird since we only created a single duplicate, as had been made sure by `self.assertEqual(2, ShiftTemplate.objects.count())`. I think it would be clearer if we fetched the new template and tested just that one.